Add alias to  service to make it accessible

Schrank opened the pull request:
### 1. Why is this change necessary?
Following the tutorial results in:

    In Container.php line 269:

    The "Shopware\Core\Framework\DataAbstractionLayer\Indexing\EntityIndexerRegistry" service or alias has been removed or
    inlined when the container was compiled. You should either make it public, or stop using the container directly and use
     dependency injection instead.

### 2. What does this change do, exactly?
To fix the problem we alias the service and make the alias public

shyim commented:
Hey @mkreusch ,
can you please sign the CLA?
Thanks! :blue_heart:

shyim commented:
Hey @mkreusch,
unfortunately this has been already fixed in the latest version https://github.com/shopware/shopware/blob/5.5/engine/Shopware/Bundle/SearchBundleDBAL/FacetHandler/ProductAttributeFacetHandler.php#L186

Thanks anyways for your contribution :blue_heart:

Schrank committed:
Add alias to  service to make it accessable

Following the tutorial results in:

    In Container.php line 269:

    The "Shopware\Core\Framework\DataAbstractionLayer\Indexing\EntityIndexerRegistry" service or alias has been removed or
    inlined when the container was compiled. You should either make it public, or stop using the container directly and use
     dependency injection instead.

to fix the problem we alias the service and make the alias public

mitelg commented:
hey @Schrank 

I don't know an what version you are, but the service is already public https://github.com/shopware/platform/blob/master/src/Core/Framework/DependencyInjection/data-abstraction-layer.xml#L556

This will be part of the upcoming 6.3.1.0 release

Schrank commented:
Damn it, time to update 🙈  .. 6.2.xx
